src/core/PrettyPrinter.js

```diff
@@ -210,6 +212,13 @@ getJasmineRequireObj().pp = function(j$) {
     this.append(' })');
   };
 
+  StringPrettyPrinter.prototype.emitTypedArray = function(obj) {
+    var constructorName = j$.fnNameFor(obj.constructor);
+    var stringifiedArray = Array.prototype.join.call(obj);
```

Comment:
Typed arrays should also respect the `MAX_PRETTY_PRINT_ARRAY_LENGTH` in the same way that normal arrays do.